src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/ReimbursementRequestDetailsView.tsx

```diff
@@ -115,7 +122,13 @@ const ReimbursementRequestDetailsView: React.FC<ReimbursementRequestDetailsViewP
             <VerticalDetailDisplay label="Sabo Number" content={`${undefinedPipe(reimbursementRequest.saboId)}`} />
           </Grid>
           <Grid item sm={6} xs={12}>
-            <VerticalDetailDisplay label="Refund Source" content={`${reimbursementRequest.account}`} />
+            <VerticalDetailDisplay
+              label="Refund Source"
+              content={`${codeAndRefundSourceName(
```

Comment:
You can get rid of the `${ because the codeAndRefundSourceName function already formats the number into a string/text